src/ring_theory/noetherian.lean

```diff
@@ -427,6 +427,25 @@ begin
   { rintro (rfl|rfl); simp [is_idempotent_elem] }
 end
 
+lemma exists_radical_pow_le_of_fg {R : Type*} [comm_semiring R] (I : ideal R) (h : I.radical.fg) :
```

Comment:
Hmm, just realized this could use dot notation for `ideal.fg`

Reply:
I think the dot notation is not that nessecary and intuitive since the condition is on `I.radical` instead.

Reply:
OK, currently the dot notation works for `ideal`, which is also good!